ext/socket/raddrinfo.c

```diff
@@ -732,12 +737,17 @@ rb_getnameinfo(const struct sockaddr *sa, socklen_t salen,
 #endif
 
     pthread_t th;
-    if (do_pthread_create(&th, 0, do_getnameinfo, arg) != 0) {
+    if (do_pthread_create(&th, &attr, do_getnameinfo, arg) != 0) {
```

Comment:
I believe this was intended to be used when creating the thread, but I might be wrong.

I think this dates back to when we only did `pthread_create` here. i.e. https://github.com/ruby/ruby/commit/3dc311bdc8badb680267f5a10e0c467ddd9dfe4c

If this isn't needed, I think we can avoid calling `pthread_attr_init` all together.

CC: @mame